Integration test for uploading an XPI to DevHub is flaky

The CircleCI integration tests for uploading an XPI to DevHub times out a lot. We should fix it or disable the test.

Example https://circleci.com/gh/mozilla/addons-frontend/8848?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

```
______ ERROR at setup of test_devhub_addon_upload[Resolution: 1080x1920] _______
...
E       TimeoutException: Message:
```

--- a/tests/ui/run-functional-tests.sh
+++ b/tests/ui/run-functional-tests.sh
@@ -20,4 +20,5 @@ docker-compose -f addons-server/docker-compose.yml -f tests/ui/docker-compose.fu
 docker-compose -f addons-server/docker-compose.yml -f tests/ui/docker-compose.functional-tests.yml restart worker
 docker-compose -f addons-server/docker-compose.yml -f tests/ui/docker-compose.functional-tests.yml exec web make -f Makefile-docker update_deps
 docker-compose -f addons-server/docker-compose.yml -f tests/ui/docker-compose.functional-tests.yml restart web
-docker-compose -f addons-server/docker-compose.yml -f tests/ui/docker-compose.functional-tests.yml exec --user root selenium-firefox tox -e ui-tests
+sudo chown -R circleci:circleci .
+docker-compose -f addons-server/docker-compose.yml -f tests/ui/docker-compose.functional-tests.yml exec --user cirlceci selenium-firefox tox -e ui-tests

--- a/tests/ui/run-functional-tests.sh
+++ b/tests/ui/run-functional-tests.sh
@@ -20,6 +20,6 @@ docker-compose -f addons-server/docker-compose.yml -f tests/ui/docker-compose.fu
 docker-compose -f addons-server/docker-compose.yml -f tests/ui/docker-compose.functional-tests.yml restart worker
 docker-compose -f addons-server/docker-compose.yml -f tests/ui/docker-compose.functional-tests.yml exec web make -f Makefile-docker update_deps
 docker-compose -f addons-server/docker-compose.yml -f tests/ui/docker-compose.functional-tests.yml restart web
-# This is done because cirlceci has permissions do not allow the user `circleci` to remove or add files created by the root, even if it is part of the root group.
+# This is done because cirlceci has permissions to not allow the user `circleci` to remove or add files created by the root, even if it is part of the root group.
 sudo chown -R circleci:circleci .
 docker-compose -f addons-server/docker-compose.yml -f tests/ui/docker-compose.functional-tests.yml exec --user cirlceci selenium-firefox tox -e ui-tests

--- a/tests/ui/run-functional-tests.sh
+++ b/tests/ui/run-functional-tests.sh
@@ -20,5 +20,6 @@ docker-compose -f addons-server/docker-compose.yml -f tests/ui/docker-compose.fu
 docker-compose -f addons-server/docker-compose.yml -f tests/ui/docker-compose.functional-tests.yml restart worker
 docker-compose -f addons-server/docker-compose.yml -f tests/ui/docker-compose.functional-tests.yml exec web make -f Makefile-docker update_deps
 docker-compose -f addons-server/docker-compose.yml -f tests/ui/docker-compose.functional-tests.yml restart web
+# This is done because cirlceci has permissions do not allow the user `circleci` to remove or add files created by the root, even if it is part of the root group.
 sudo chown -R circleci:circleci .
 docker-compose -f addons-server/docker-compose.yml -f tests/ui/docker-compose.functional-tests.yml exec --user cirlceci selenium-firefox tox -e ui-tests

